Its.Log.UnitTests/Its.Log.UnitTests.csproj

```diff
@@ -1,18 +1,12 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="12.0">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net462</TargetFramework>
```

Comment:
This should likely also target `netcoreapp2.1` or `netcoreapp2.2`.

Reply:
Should say:

`<TargetFrameworks>net462;netcoreapp2.1</TargetFrameworks>`

Reply:
I got the test project working with .NET Core 2.2 but one of the tests is not working (ToLogString_can_be_made_to_output_JSON in JsonSerializationTests). Somehow in .NET Core 2.2 the JsonSerializer cannot serialize the System.IO.FileInfo object. I will try to dig deeper.

Also I am having trouble get targeting 2 frameworks working. Can I just target .NET Core 2.2?

Reply:
I changed the ToLogString_can_be_made_to_output_JSON test so it works for .NET Core 2.2. I guess .NET Core 2.2's FileInfo is just not serializable by Newtonsoft. I use LogEntry to test ToLogString instead.